### PR TITLE
Fix create_composite_with_component_ids

### PR DESCRIPTION
**Pull request by odovzhenko**
Close https://github.com/SatelliteQE/robottelo/issues/5396

```
(sat-6.3.0) odovz@bueno:~/projects/robottelo$ pytest -v tests/foreman/cli/test_contentview.py::ContentViewTestCase::test_positive_create_composite_with_component_ids tests/foreman/cli/test_contentview.py::ContentViewTestCase::test_negative_create_composite_with_component_ids
======================================================================================= test session starts ========================================================================================
platform linux2 -- Python 2.7.12, pytest-3.2.3, py-1.4.34, pluggy-0.4.0 -- /home/odovz/venv/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/odovz/projects/robottelo, inifile:
plugins: xdist-1.20.1, services-1.2.1, mock-1.6.3, forked-0.2, cov-2.5.1
collected 2 items                                                                                                                                                                                   
2017-11-21 16:56:43 - conftest - DEBUG - BZ deselect is disabled in settings


tests/foreman/cli/test_contentview.py::ContentViewTestCase::test_positive_create_composite_with_component_ids <- robottelo/decorators/__init__.py PASSED
tests/foreman/cli/test_contentview.py::ContentViewTestCase::test_negative_create_composite_with_component_ids <- robottelo/decorators/__init__.py PASSED

==================================================================================== 2 passed in 102.38 seconds ====================================================================================
```

### CODE REVIEW

File: tests/foreman/cli/test_contentview.py [hunk 1]
```diff
@@ -1068,28 +1068,30 @@ def test_positive_remove_component_by_name(self):
         self.assertEqual(len(comp_cv['components']), 0)
 
     @tier2
-    @skip_if_bug_open('bugzilla', 1487265)
     @run_only_on('sat')
     def test_positive_create_composite_with_component_ids(self):
-        """create a composite content view with a component_ids option
+        """Create a composite content view with a component_ids option contains
+            ids from different components
```
Comment (ldjebran) [inline tests/foreman/cli/test_contentview.py]: Create a composite content view with a component_ids option which ids are from different content views components

File: tests/foreman/cli/test_contentview.py [hunk 2]
```diff
@@ -1068,28 +1068,30 @@ def test_positive_remove_component_by_name(self):
...
 
         :id: 6d4b94da-258d-4690-a5b6-bacaf6b1671a
 
-        :expectedresults: Composite content view component ids are similar to
-            the nested content view versions ids
-
-        :BZ: 1487265
+        :expectedresults: Composite content view contains component ids from
+            different components
```
Comment (ldjebran) [inline tests/foreman/cli/test_contentview.py]: Composite content view created

File: tests/foreman/cli/test_contentview.py [hunk 3]
```diff
@@ -1106,6 +1108,42 @@ def test_positive_create_composite_with_component_ids(self):
             'the input values',
         )
 
+    @tier2
+    @skip_if_bug_open('bugzilla', 1487265)
+    @run_only_on('sat')
+    def test_negative_create_composite_with_component_ids(self):
+        """Try create a composite content view with a component_ids option
+            contains ids from same component
```
Comment (ldjebran) [inline tests/foreman/cli/test_contentview.py]: Attempt to create a composite content view with a component_ids option which ids are from the same content view

File: tests/foreman/cli/test_contentview.py [hunk 4]
```diff
@@ -1106,6 +1108,42 @@ def test_positive_create_composite_with_component_ids(self):
...
+
+        :id: 27ea1f41-44b7-40f0-8990-a4aed09b06b2
+
+        :expectedresults: User cannot create a composite content view that
```
Comment (ldjebran) [inline tests/foreman/cli/test_contentview.py]: Composite content view not created

File: tests/foreman/cli/test_contentview.py [hunk 5]
```diff
@@ -1068,28 +1068,30 @@ def test_positive_remove_component_by_name(self):
...
-        """create a composite content view with a component_ids option
+        """Create a composite content view with a component_ids option which
+            ids are from different content views components
 
         :id: 6d4b94da-258d-4690-a5b6-bacaf6b1671a
 
-        :expectedresults: Composite content view component ids are similar to
-            the nested content view versions ids
-
-        :BZ: 1487265
```
Comment (oshtaier) [inline tests/foreman/cli/test_contentview.py]: We remove only decorators, but keeping BZ information in docstring for history

File: tests/foreman/cli/test_contentview.py [hunk 6]
```diff
@@ -1106,6 +1108,42 @@ def test_positive_create_composite_with_component_ids(self):
             'the input values',
         )
 
+    @tier2
+    @skip_if_bug_open('bugzilla', 1487265)
```
Comment (oshtaier) [inline tests/foreman/cli/test_contentview.py]: Defect is already in VERIFIED state, so no reason to add that decorator

File: tests/foreman/cli/test_contentview.py (hunk 5 above)
Comment (oshtaier) [reply tests/foreman/cli/test_contentview.py]: oh, it is another test case. Please, ignore that comment

File: tests/foreman/cli/test_contentview.py (hunk 6 above)
Comment (oshtaier) [reply tests/foreman/cli/test_contentview.py]: but keep the ID in docstring

**Comment by odovzhenko** [comment]
@ldjebran and @oshtaier 
Changes addressed.

File: tests/foreman/cli/test_contentview.py [hunk 7]
```diff
@@ -1068,28 +1068,31 @@ def test_positive_remove_component_by_name(self):
         self.assertEqual(len(comp_cv['components']), 0)
 
     @tier2
-    @skip_if_bug_open('bugzilla', 1487265)
     @run_only_on('sat')
     def test_positive_create_composite_with_component_ids(self):
-        """create a composite content view with a component_ids option
+        """Create a composite content view with a component_ids option which
+            ids are from different content views
```
Comment (oshtaier) [inline tests/foreman/cli/test_contentview.py]: I know, it is nitpick, but it should be
```
"""Create a composite content view with a component_ids option which
ids are from different content views
```
and in next test case - the same

**Pull request merged**